files/class-vip-filesystem-api-cache.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace Automattic\VIP\Files;
+
+use WP_Error;
+
+
+class API_Cache {
```

Comment:
Would be nice to add some basic tests for this new class.

Reply:
I already did. Perhaps you missed it?

Reply:
Ah, sorry. Forgot to refresh the page :(

Reply:
Test looks good! 💯